android/src/main/java/com/appfolio/uploader/UploaderModule.kt

```diff
@@ -200,16 +202,26 @@ class UploaderModule(val reactContext: ReactApplicationContext) : ReactContextBa
         ModifiedBinaryUploadRequest(this.reactApplicationContext, url!!, limitNetwork)
                 .setFileToUpload(filePath!!)
       } else {
-        if (!options.hasKey("field")) {
-          promise.reject(java.lang.IllegalArgumentException("field is required field for multipart type."))
-          return
+        if (parts != null && parts.size() < 1) {
```

Comment:
Should this be `parts == null || parts.size() < 1`?

Reply:
Oh yea either way. The null check is so I don't get an exception when checking parts.size() but there's a null check in the app before it gets here to protect against it.

Reply:
@kyleweisel is correct. Change it to his line.

Reply:
I updated it